### DISCUSSION TITLE
Postman Collection added to Project

### DISCUSSION THREAD

**Discussion by jensenkd** [jensenkd/plex-api#68] [Ideas]
I'd like to find out if others have seen Postman collections added to github projects elsewhere?

It would be nice to have a full collection of Plex API methods to reference and test out new functions.

Thoughts?

**Comment by seertenedos** [comment]
i really have not seen much. So far the best i have found is https://github.com/Arcanemagus/plex-api/wiki/Plex-Web-API-Overview but most of the time i learn more from the developer tools in the browser.

i really wanted an API to edit the dates on a series/season/episode like the data added and last watched but i don't think it exists

**Reply by seertenedos** [reply]
Maybe ask on the plex forums as well or look at the API implementations in some of the other languages?
